hazelcast/src/main/java/com/hazelcast/concurrent/lock/LockStore.java

```diff
@@ -157,7 +157,8 @@
      * waiters/signals/expired operations.
      *
      * @param dataKey the lock key
-     * @return if the key was unlocked or is already unlocked
+     * @return if the key was unlocked or is already unlocked but still in the store. Returns false only if the key is
```

Comment:
Can you rephrase this a bit to something like `@return {@code false} if there is no lock for the given {@code key} and {@code true} otherwise`